File: Previous Contest/3498. Reverse Degree of a String/Solution.java
class Solution {
    public int reverseDegree(String s) {
        // N : len(nums)
        // Time Complexity: O(N)
        // Space Complexity: O(1)
        int n = s.length();
        int reverseDegree = 0;
        for (int i = 0; i < n; i++) {
            reverseDegree += ('z' - s.charAt(i) + 1) * (i + 1);
        }
        return reverseDegree;
    }
}

File: Previous Contest/3498. Reverse Degree of a String/Solution.py
class Solution:
    def reverseDegree(self, s: str) -> int:
        # N : len(nums)
        # Time Complexity: O(N)
        # Space Complexity: O(1)
        n = len(s)
        reverseDegree = 0
        for i in range(n):
            reverseDegree += (ord('z') - ord(s[i]) + 1) * (i + 1)
        return reverseDegree